Source/WebKit/UIProcess/API/Cocoa/NSAttributedString.h

```diff
@@ -72,6 +72,21 @@ typedef void (^NSAttributedStringCompletionHandler)(NSAttributedString * _Nullab
 + (void)loadFromHTMLWithRequest:(NSURLRequest *)request options:(NSDictionary<NSAttributedStringDocumentReadingOptionKey, id> *)options completionHandler:(NSAttributedStringCompletionHandler)completionHandler
     NS_SWIFT_NAME(loadFromHTML(request:options:completionHandler:)) WK_SWIFT_ASYNC_NAME(fromHTML(request:options:)) WK_API_AVAILABLE(macos(10.15), ios(13.0));
 
+/*!
+ @abstract Loads an HTML URL request and converts the contents into an attributed string.
+ @param request The request specifying the URL to load.
+ @param options Document attributes for interpreting the document contents.
+ NSTextSizeMultiplierDocumentOption and NSTimeoutDocumentOption are supported option keys.
+ @param queue The queue to call back on
+ @param completionHandler A block to invoke when the operation completes or fails.
+ @discussion The completionHandler is passed the attributed string result along with any
+ document-level attributes, or an error.
+*/
++ (void)loadFromHTMLWithRequest:(NSURLRequest *)request options:(NSDictionary<NSAttributedStringDocumentReadingOptionKey, id> *)options
+    queue:(dispatch_queue_t _Nullable)queue
+    completionHandler:(NSAttributedStringCompletionHandler)completionHandler
+    NS_SWIFT_NAME(loadFromHTML(request:options:queue:completionHandler:)) WK_SWIFT_ASYNC_NAME(fromHTML(request:options:queue:)) WK_API_AVAILABLE(macos(10.15), ios(13.0));
```

Comment:
API availability should be WK_MAC_TBA and WK_IOS_TBA, not these long-shipped versions.